Tests/test_color_lut.py

```diff
@@ -0,0 +1,129 @@
+from __future__ import division
+from helper import unittest, PillowTestCase
+
+import time
```

Comment:
Unused import. Although you may tell me to save these comments for when this is less of a work-in-progress.